src/main/java/uk/gov/justice/digital/delius/data/api/Nsi.java

```diff
@@ -18,6 +19,7 @@
     private KeyValue nsiSubType;
     private Requirement requirement;
     private KeyValue nsiStatus;
+    private LocalDateTime statusDate;
```

Comment:
I don't have a strong opinion: do you think it should be called `statusDatetime` so it is obvious that this has a time element? Do we have a precedent for this when mixing types?

Reply:
I’ll rename the field. In AddressAssessment, there are 3 LocalDateTime fields, 2 have the suffix “Datetime” and one has “Date”. ApprovedPremisesReferral has a similar mix.